data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -8,6 +8,8 @@
 GEOID_TRACT_FIELD = "GEOID10_TRACT"
 STATE_FIELD = "State/Territory"
 COUNTY_FIELD = "County Name"
+REGION_FIELD = "Region"
```

Comment:
Nit: Region actually has a specific census meaning that we use in the comparison tool (https://www2.census.gov/geo/pdfs/maps-data/maps/reference/us_regdiv.pdf), is there another name for this? Nothing obvious comes to mind for me. Could even make it very UI-specific (since that's what this is) and call it something like USER_INTERFACE_EXPERIENCE (I don't love that, but you get the idea).

Reply:
yes! this makes sense